apps/backend/src/users/pantry.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
```

Comment:
we can move this file (pantry.entity.ts) to another folder, pantries, for consistency and import it from there as needed but the content of this is good!

Reply:
Changes applied accordingly